python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -99,6 +99,8 @@ def __init__(self, num_processes: int = 1,
                     "Fall back to regular pytorch.")
             use_ipex = False
 
+        self.use_ipex = use_ipex
+
```

Comment:
let's use `self.use_ipex` for rest of this function

Reply:
Done